test/system/505-networking-pasta.bats

```diff
@@ -434,43 +434,43 @@ function pasta_test_do() {
 
 ### DNS ########################################################################
 
-@test "External resolver, IPv4" {
-    skip_if_no_ipv4 "IPv4 not routable on the host"
-
-    run_podman '?' run --rm --net=pasta $IMAGE nslookup 127.0.0.1
-
-    assert "$output" =~ "1.0.0.127.in-addr.arpa" \
-           "127.0.0.1 not resolved"
-}
-
-@test "External resolver, IPv6" {
-    skip_if_no_ipv6 "IPv6 not routable on the host"
-
-    run_podman '?' run --rm --net=pasta $IMAGE nslookup ::1
-
-    assert "$output" =~ "1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa" \
-           "::1 not resolved"
+@test "Basic nameserver lookup" {
+    run_podman run --rm --net=pasta $IMAGE nslookup l.root-servers.net
```

Comment:
That makes sense, but I think its out of scope for this series.  If you could do that as a followup, that would be great.  I think it will be a bit faster for you since you'll be more familiar with the podman codebase at large.